packages/blade/src/hooks/useControllable.ts

```diff
@@ -1,8 +1,8 @@
 /* eslint-disable @typescript-eslint/explicit-module-boundary-types */
 /* eslint-disable @typescript-eslint/explicit-function-return-type */
 /* eslint-disable react-hooks/exhaustive-deps */
+import { isFunction } from 'lodash';
```

Comment:
Should we import directly? Would this code split? I remember lodash having issues with code splitting earlier.

Reply:
Right will change all the instances all over the codebase. found 3